_scss/components/_header.scss

```diff
@@ -101,3 +109,7 @@
   @include u-text("no-uppercase", "primary");
   @include u-width(10);
 }
+
+#search_form {
```

Comment:
Let's avoid IDs for selectors let's use a class that targets search in header **or** add a new class `site-search`.

Reply:
Replaced with a class